### ISSUE TITLE
Scrubbing check during recording video buffering metric

### ISSUE THREAD

**Issue by subhendusethi** [spodlecki/videojs-event-tracking#9]
Hi.

My question was related to the following behavior:

- Why do we have the check on scrubbing for recording buffering metric? The reason I am asking is that whenever I am seeking the video to a considerable timestamp forward or backward the video player is waiting/buffering and because of [this](https://github.com/spodlecki/videojs-event-tracking/blob/master/src/tracking/buffering.js#L52) check I am losing on that important data point.

Regards,
Subhendu

**Comment by spodlecki** [comment]
This is actually something I debated for a while when writing this portion. I started looking for what I wanted to answer in the metrics. Specifically for buffering, I was largely answering the questions: 

> How many users while playing a video, experienced buffering?
> How long were these users in a buffering state?

Users who used the scrub bar are basically reloading the video with intent, so those users didn't matter in that analysis (for my use-case).

--------

I can certainly see your use-case. I am open to receiving a Pull Request that would solve your issue. I'd want backwards compatibility -- so possibly adding an option..

```javascript
videojs('videodomid', {
    ...videoConfig,
    plugins: {
      eventTracking: {
        bufferingConfig: { includeScrub: false }
      }
    }
});
```
